packages/dev/core/src/Physics/v2/physicsBody.ts

```diff
@@ -338,14 +375,18 @@ export class PhysicsBody {
      * Filtering by body is inefficient. It's more preferable to register a collision callback for the entire world
      * and do the filtering on the user side.
      */
-    public registerOnCollide(func: (collider: PhysicsBody, collidedAgainst: PhysicsBody, point: Nullable<Vector3>) => void): void {
+    public registerOnCollide(

```

Comment:
params are missing in the doc. @RaananW  I wonder why the doc check did not caught them ?

Reply:
this code has been discussed in this PR. see previous comments.